core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/impl/BasePackageServiceImpl.java

```diff
@@ -142,15 +147,21 @@ public boolean isEnableStackTrace() {
      */
     @Override
     public PackageInfo getPackageInfo(final ResourceResolver resourceResolver, final PackageModel packageModel) {
```

Comment:
Is it possible to pass switch value in the model and base logic on boolean value? Also it would be nice to include this flag to package info

Reply:
added switch flag in package model and package info